src/main/util.ts

```diff
@@ -84,3 +85,32 @@ export const isDebug = () => process.env.NODE_ENV === 'development' || process.e
 
 export const appdataPath = app.getPath('userData');
 export const logsPath = app.getPath('logs');
+
+
+export const elevatedPSCommand = (command: string): Promise<string> => {
+    return new Promise((resolve, reject) => {
+        exec(`powershell -Command "& {Start-Process powershell -windowstyle hidden -Verb RunAs -ArgumentList '-Command ${command}'}"`, (error: any, stdout: any, stderr: any) => {
+            if (error) {
+                reject(error);
+            }
+            if (stderr) {
+                reject(stderr);
+            }
+            resolve(stdout);
+        });
+    });
+}
+
+export const psCommand = (command: string): Promise<string> => {
+    return new Promise((resolve, reject) => {
+        exec(`powershell -Command "${command.replaceAll('"', '\\"')}"`, (error: any, stdout: any, stderr: any) => {
```

Comment:
It has worked up to this point.  I think because it is inside of a string, it behaves differently

Reply:
Also, the only reason I had to add the escaping is because some non-elevated ps command needed escaping (hence why this one got escaped and the other one didn't)